lib/MicrocopyController.ts

```diff
@@ -0,0 +1,38 @@
+import type { ReactiveController, ReactiveControllerHost } from 'lit';
+
+export class MicrocopyController extends Map<string, string> implements ReactiveController {
+  constructor(private host: ReactiveControllerHost, obj: Record<string, string>) {
+    super(Object.entries(obj));
+  }
+
+  get(key: string) {
+    return super.get(key) ?? key;
+  }
+
+  set(key: string, value: string) {
+    super.set(key, value);
+    this.host.requestUpdate();
```

Comment:
@bennypowers It looks like set() runs before `this.host` is available.
https://lit.dev/playground/#gist=16ef8ba7241a4ddec2f9a4021d6b42c5

Reply:
we could either skip the initial set like you did in the playground, or defer the initial join until after updateComplete

Reply:
can fix this in a followup if it's still relevant